pkg/cli/admin/release/new.go

```diff
@@ -297,6 +299,7 @@ type CincinnatiMetadata struct {
 
 	Version  string   `json:"version"`
 	Previous []string `json:"previous"`
+	Next []string `json:"next"`
```

Comment:
`omitempty`?  It's a useful safety-valve, but there's enough risk in recommending vB update to an older vA this way, that I dunno if we want `"next": []` in metadata reminding folks that this is an option ;)